test/testUtils.js

```diff
@@ -30,6 +30,26 @@ export function mockResponse(data, meta, warnings, status = 200) {
       },
     },
   })
+}
+
+export function mockMatchingURLResponse(
+  urlOrRegExp,
```

Comment:
Name is inferred from the naming that Moxios uses.
See: https://github.com/axios/moxios/blob/master/index.js#L316